.github/workflows/release.yml

```
@@ -0,0 +1,220 @@
+        shell: bash
+      
+      - name: Package binary
+        run: |
+          mkdir -p dist
+          cp cli/target/${{ matrix.target }}/release/prompts${{ matrix.binary-suffix }} dist/
+          cp README.md dist/
+          cp -r scripts dist/
+        shell: bash
+      
+      - name: Create archive
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2035:info:2:92: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)